lib/cdo/rack/upgrade_insecure_requests.rb

```diff
@@ -50,7 +50,7 @@ def call(env)
             "style-src 'self' https: 'unsafe-inline'",
             "img-src 'self' https: data: blob:",
             "font-src 'self' https: data:",
-            "connect-src 'self' https: https://api.pusherapp.com wss://ws.pusherapp.com wss://*.firebaseio.com http://localhost:8080 https://curriculum.code.org/",
+            "connect-src 'self' https: https://api.pusherapp.com wss://ws.pusherapp.com wss://*.firebaseio.com http://localhost:8080 https://curriculum.code.org/ wss://*.code.org",
```

Comment:
TODO: Extract this to a configuration setting declared and set in config.yml.erb